src/js/components/ChooseProjectModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { Tabs, List, Avatar, Modal, Button, Space, Typography } from 'antd';
+import { useAppSelector } from '@/hooks';
+import { Link, useLocation } from 'react-router-dom';
+
+const ChooseProjectModal: React.FC<ChooseProjectModalProps> = ({ isModalOpen, setIsModalOpen }) => {
+  const location = useLocation();
+  const { projects, selectedScope } = useAppSelector((state) => state.metadata);
+  const [selectedProject, setSelectedProject] = useState(selectedScope.project ?? projects[0].identifier);
+
+  const baseURL = '/' + location.pathname.split('/')[1];
+
+  const closeModal = () => setIsModalOpen(false);
+
+  return (
+    <Modal
+      title="Choose View"
+      open={isModalOpen}
+      onCancel={closeModal}
+      width={800}
+      footer={
+        <Space>
+          <Button key="1" onClick={closeModal}>
+            Cancel
+          </Button>
+          <Link to={baseURL} key="2">
+            <Button>Select Default View</Button>
+          </Link>
+        </Space>
+      }
+    >
+      <Tabs
+        tabPosition="left"
+        activeKey={selectedProject}
+        onChange={(key) => setSelectedProject(key)}
+        items={projects.map(({ identifier, title, datasets, description }) => {
```

Comment:
I'm not a big fan of the way these are rendered - the visual hierarchy is unclear, as is the "cancel" button's purpose and the "select default view" button.

Reply:
How about this: 
![image](https://github.com/user-attachments/assets/ed68004b-c4ac-4fa6-8fd9-0c00a77a1c65)

Reply:
As a sometimes confused user, I'm still a bit puzzled at the user actions needed to choose between viewing a whole project and just viewing a particular dataset. Intuitively you'd think they would be more similar.